stable/snap/snapcraft.yaml

```diff
@@ -61,6 +61,19 @@ parts:
       mkdir -p $SNAPCRAFT_PART_INSTALL/opt/powershell
       tar zxf powershell.tar.gz -C $SNAPCRAFT_PART_INSTALL/opt/powershell
       chmod +x $SNAPCRAFT_PART_INSTALL/opt/powershell/pwsh
+      echo "fixing symlinks"
```

Comment:
This issue existed in the preview.  The stable is not  being affected for whatever reason